app/assets/stylesheets/application.css.scss

```diff
@@ -24,7 +24,7 @@ $navbar-default-color: white;
 $navbar-default-link-color: white;
 $navbar-default-link-hover-color: #DDDDDD;
 $table-bg-accent: #E8E8E8;
-
+@import "bootstrap-sprockets";
```

Comment:
I know this line is to fix glyphicons so leave it. Just in case we need them for something.